hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/tenant/TenantCreateHandler.java

```diff
@@ -33,26 +32,19 @@
     description = "Create one or more tenants")
 public class TenantCreateHandler extends TenantHandler {
 
-  @CommandLine.Spec
-  private CommandLine.Model.CommandSpec spec;
-
-  @CommandLine.Parameters(description = "List of tenant names")
+  @CommandLine.Parameters(description = "List of tenant names", arity = "1..")
   private List<String> tenants = new ArrayList<>();
 
   @Override
   protected void execute(OzoneClient client, OzoneAddress address) {
-    if (tenants.size() > 0) {
-      for (String tenantName : tenants) {
-        try {
-          client.getObjectStore().createTenant(tenantName);
-          out().println("Created tenant '" + tenantName + "'.");
-        } catch (IOException e) {
-          err().println("Failed to create tenant '" + tenantName + "': " +
-              e.getMessage());
-        }
+    for (String tenantId : tenants) {
```

Comment:
Do we have other CLIs like this that issue multiple OM requests? If nobody else is doing this, I would say we avoid it. It's confusing to the user what state is created if one of the multiple requests fails.

Reply:
I think I will file a jira to remove the loop for create/delete tenant CLI.

But this might actually have value for assign user to tenant? i.e. An admin batch assigning users to a tenant without waiting for seconds every time the Ozone CLI client's JVM boots up.

Reply:
Removed loop in `CreateTenant` and `DeleteTenant`. Might remove others in another refactoring jira.